egs2/ksponspeech/asr1/run.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+train_set="train"
+valid_set="dev"
+test_sets="eval_clean eval_other"
+asr_config="conf/train_asr_conformer8_n_fft512_hop_length256.yaml"
+lm_config="conf/train_lm_transformer3.yaml"
```

Comment:
change them to use symlinks according to the changes that I suggested.

Reply:
Fixed as suggested.